### PR TITLE
Fixed vague, inconsistent and unclear texts

### PR DESCRIPTION
**Pull request by dn0**
Fixed vague, inconsistent and unclear texts:
 
- Added BIOS configuration section into prerequisites section
 - Renamed logical networks -> virtual networks
 - Added missing information about required disk space

### CODE REVIEW

File: user-guide/install/before.rst [hunk 1]
```diff
@@ -120,6 +122,32 @@ Below is an example port map for interconnection of two *Danube Cloud* nodes (on
...
+BIOS Configuration
+##################
+
+The following settings should be configured in BIOS configuration of your compute node (if available):
+
+- Enable hardware virtualization (KVM) support.
+
+    .. warning:: Hardware virtualization (KVM) support must enabled on the head node.
+
+- Disable NUMA/Node interleaving. If not disabled the follow message may appear during boot time:
```
Comment (secult) [inline user-guide/install/before.rst]: If not disabled, the following message may appear during boot time:

File: user-guide/install/before.rst [hunk 2]
```diff
@@ -120,6 +122,32 @@ Below is an example port map for interconnection of two *Danube Cloud* nodes (on
...
+
+    .. code-block:: text
+
+        WARNING: Couldn't read ACPI SRAT table from BIOS. lgrp support will be limited to one group.
+
+- Disable CPU C-States.
+
+    .. note:: Some Intel® processors, which are using the C-States feature, can cause an error that may seriously endanger the run of a working compute node. The error is treated in the system, but you are advised to disable C-States in the BIOS configuration.
```
Comment (secult) [inline user-guide/install/before.rst]: More information may be beneficial:

- under what circumstances the error happens
- in what way the error is handled in the system
- maybe there should be some external link on the issue

File: user-guide/install/before.rst [hunk 3]
```diff
@@ -120,6 +122,32 @@ Below is an example port map for interconnection of two *Danube Cloud* nodes (on
...
+.. _bios:
+
+BIOS Configuration
+##################
+
+The following settings should be configured in BIOS configuration of your compute node (if available):
+
+- Enable hardware virtualization (KVM) support.
+
+    .. warning:: Hardware virtualization (KVM) support must enabled on the head node.
```
Comment (YanChii) [inline user-guide/install/before.rst]: On all nodes, not only on head node.

File: user-guide/install/before.rst (hunk 1 above)
Comment (YanChii) [inline user-guide/install/before.rst]: typo: "following"
First thought: I don't want to disable NUMA.
Explaination hints: You need to disable NUMA/Node Interleaving in your BIOS to get the ACPI SRAT unless the BIOS has a specific option to enable ACPI SRAT.  You really aren't disabling NUMA, you are allowing the OS to handle it.
Source: https://blog.alainodea.com/en/article/445/smartos-on-multi-socket-and-the-acpi-srat

File: user-guide/install/before.rst (hunk 2 above)
Comment (YanChii) [inline user-guide/install/before.rst]: Some Intel® processors that are using the C-States feature, can cause an error that may seriously endanger the correct run of a compute node. The error is treated in the system but you are advised to disable C-States in the BIOS configuration.

File: user-guide/gui/datacenters/networks.rst [hunk 4]
```diff
@@ -45,14 +45,18 @@ Managing a Network
...
+
+.. note:: A virtual network is connected to the physical network interfaces on compute nodes via NIC tags. :ref:`NIC tags must be configured on every compute node in advance <network>`.
+
 Advanced Network Settings
 -------------------------
 
     .. image:: img/add_network_details.png
 
-* **NIC Tag** - NIC tag or device name configured on a compute node. One of: *admin*, *internal*, *external* and *storage*.
+* **NIC Tag** - :ref:`NIC tag <network_nictag>` or device name configured on a compute node. One of: *admin*, *internal*, *external* and *storage*.
```
Comment (YanChii) [inline user-guide/gui/datacenters/networks.rst]: Can you really use device name directly instead of nic tag?

**Review by YanChii — CHANGES_REQUESTED** [review]
A few comments in-line.

**Review by secult — APPROVED** [review]

**Review by YanChii — APPROVED** [review]

**Pull request merged**